cmd/backup-operator/README.md

```diff
@@ -27,9 +24,10 @@ NAME                                    READY     STATUS    RESTARTS   AGE
 etcd-backup-operator-1102130733-hhgt7   1/1       Running   0          3s
 ```
 
-### Create AWS Secret
+### Setup AWS Secret
 
-Create a Kubernetes secret that contains aws config/credential; etcd backup operator uses the secret to gain access to S3 in order to save the etcd snapshot.
+Create a Kubernetes secret that contains aws config/credential;
+the secret will be used later to save etcd backup into S3.
```

Comment:
don't use passive voice. the best practice is to use active voice. 
maybe change the latter part to:
`etcd backup operator uses the secret to gain access to S3 in order to save the etcd snapshot.` -> 
`etcd backup operator uses the secret to save the etcd snapshot to S3`?

Reply:
We don't need to mention backup operator. It's only about the secret

Reply:
I think technical writing should be specific such as who uses what?

Reply:
It's unwise to argue how to write technical doc here.
This sentence could be more concise as changed here.
Actually it should also mention explicitly below that the CR will use secret `aws`

Reply:
anyways, just suggestions.

Reply:
Yeah. Totally understand.
We will go through docs again and again these days and make more changes along the way.
Please check the docs again as it's more streamlined now.